### ISSUE TITLE
Add configure and copy items to action-bar

### ISSUE THREAD

**Issue by nicholasrice** [microsoft/fast-dna#292]
Clicking "copy" should copy the active components code. This code will be relative to the active-framework (React, Angular).

Configure simply toggles the open-state of the configuration menu in the right pane.

**Comment by nicholasrice** [comment]
blocked by #298

**Comment by eljefe223** [comment]
Closing because no longer relevant.
